src/librustc_borrowck/diagnostics.rs

```diff
@@ -198,7 +198,7 @@ fn main() {
 ```
 "##,
 
-E0386: r##"
+/*E0386: r##"
```

Comment:
(well I guess deleting all this long commented-out diagnostic text and replacing it with one line in `register_diagnostics!` could be an attractive option...)